packages/elements/src/utils/elementIs.ts

```diff
@@ -60,4 +66,44 @@ const elementIs = {
   sequenceMedia: isSequenceMediaElement,
 };
 
-export default elementIs;
+function isDraftMediaElement(e: Draft<Element>): e is Draft<MediaElement> {
+  return 'resource' in e && Boolean(e.resource);
```

Comment:
Wouldn't that `Boolean()` check also work for `isMediaElement` et al and then we don't need all these separate functions?

Right now it's a bit repetitive and also makes `elements` depend on `immer` just for of these type guards.

Reply:
It would work yes, but the type of the function would be wrong, no? And we could probably change the dependency to be @types/immer instead?

Reply:
We could fix the type with an overload though. But that still comes with a dependency on the immer types.

Reply:
Cleaned it up a bit and simplified some usages of the filter (if you call filter with _just the guard_, TypeScript can do its magic)

Reply:
Seems to do the trick 👍  Also with the dev dependency